build/dependency

```diff
@@ -87,15 +87,19 @@ pkg_make_check() {
 
 pkg_make_install() {
     echo "- make $pkg_make_install_flags install"
-    based=$pkg_rootdir/third_party/tmp/$pkg_name-$pkg_branch_or_tag
-    adir=$based/$pkg_cross_arch
+    vspec=$pkg_branch_or_tag
+    if [ "$pkg_tip" != "" ]; then
+        vspec="$pkg_tip"
+    fi
```

Comment:
We use as version indicator the tip of the branch, if available. Elsewhere in this file there is a check by which, if the branch tip is different from the expected tip, the compile script fails. This is done to ensure that we are building with exactly the branch tip that we want and not some random commit in the branch.